docs/DeveloperGuide.md

```diff
@@ -2,36 +2,524 @@
+- Works in the field of drug distribution, such as pharmacies and doctors' offices.
+- Manages a large inventory of pharmaceuticals with varying details (expiration dates, manufacturers, storage
+  conditions).
+- Prefers desktop applications for their work.
+- Proficient in fast typing.
+- Favors typing over mouse interactions.
+- Comfortable using Command Line Interface (CLI) applications for efficiency.
+- Requires password-protected access to sensitive patient healthcare information.
 
 ### Value proposition
 
-{Describe the value proposition: what problem does it solve?}
+**Stocker** is designed to cater to the specific needs of drug distributors by offering the following benefits:
+
+1. Quick access to real-time inventory information.
+2. Efficient tracking of incoming stock.
+3. Categorization of drugs based on various labels.
+4. A prioritized list of urgently needed restocks for timely replenishment.
+5. Assistance in generating comprehensive reports for stock turnover analysis.
+6. Secure access through individual user accounts to safeguard patient healthcare data.
+7. Enhanced user experience for experienced professionals who prefer keyboard commands and CLI interactions for seamless
+   stock management.
 
 ## User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+Priorities: High (must have) - \* \* _, Medium (nice to have) - _ _, Low (unlikely to have) - _
+
+| Priority | Version | As a ...             | I want to ...                                        | So that I can ...                                           |
+|----------|---------|----------------------|------------------------------------------------------|-------------------------------------------------------------|
+| \* \* \* | v1.0    | Pharmacist           | Add drugs to track what drugs are available in stock | Reduce manual errors                                        |
+| \* \* \* | v1.0    | Pharmacist           | Remove drugs to track what are no longer in used     | Ensure compliance                                           |
+| \* \* \* | v1.0    | Receptionist         | View a list of products of that category             | Easily obtain an overview of the products                   |
+| \* \* \* | v1.0    | First-time user      | See a list of all available actions                  | Better understand how to use the application                |
+| \* \* \* | v1.0    | Inventory Manager    | Find a specific drug currently in the system         | Check up its details and quantities specifically            |
+| \* \* \* | v2.0    | user                 | Find a to-do item by name                            | Locate a to-do without having to go through the entire list |
```

Comment:
Just a small consideration to organise based on v1.0 then v2.0 so it looks more organised